index.js

```diff
@@ -390,7 +388,6 @@ function fastifyMultipart (fastify, options, done) {
 
         try {
           fieldValue = secureJSON.parse(fieldValue)
-          mimetype = 'application/json'
```

Comment:
I would keep this line to release this change as feature instead of breaking:

if users are using `mimetype === 'application/json'` and the content type is `application/json+vnd` we don't want to mutate the value

Reply:
Sure. It would also affect `application/json;charset=utf-8` which some applications incorrectly send.